lib/matplotlib/pyplot.py

```diff
@@ -1047,13 +1047,10 @@ def subplots(nrows=1, ncols=1, sharex=False, sharey=False, squeeze=True,
 
     Parameters
     ----------
-    nrows : int, optional, default: 1
-        Number of rows of the subplot grid.
+    nrows, ncols : int, optional, default: 1
```

Comment:
It is, just not for sharex and sharey, and the only reason is that I can't be bothered to fix the fact that the line is then too long…
I'll let someone else fix that.